docs/pages/development/develop-your-project.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Develop your project
```

Comment:
I got to admit I can't think of a better one, but I wish I this title tied a little better to the "Create Development Builds" step. Maybe "Using Development Builds"? I dunno, still kind of vague. My thought with the current title was that "wait, I'm already developing my project" ;-).

Not a big deal, just musing :-)

Reply:
I'm not a big fan of "develop your project" either but it existed before so I kept on using it.

I think "Use development builds" fits the context more, it helps to keep things simple and describe them for what they are for in the doc.